css/styles.css

```diff
@@ -3,6 +3,45 @@
   position: relative;
 }
 
+.appCover {
+  background-image: url('../images/cute-wallpaper-200921142328121024x768.jpeg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.coverText {
+  top: 50%;
+  left: 50%;
+  text-align: center;
+  color: white;
+}
```

Comment:
What is the purpose of this? I don't see it in your gif for the feature anywhere